Formula/subversion.rb

```diff
@@ -98,6 +98,8 @@ def install
         "b'SERF_MINOR_VERSION ([0-9]+).*'"
         s.gsub! "'SERF_PATCH_VERSION ([0-9]+)'",
         "b'SERF_PATCH_VERSION ([0-9]+)'"
+        s.gsub! "variables=opts,",
+        "variables=opts, RPATHPREFIX = '-Wl,-rpath,',"
```

Comment:
Thanks, been meaning to look into this! Have you been able to reproduce this? I ask but I'm slightly concerned if it's a general Scons issue rather than serf-specific, but haven't been able to reproduce the issue in the first place.